examples/flutter_view/integration_test/macos_test.dart

```diff
@@ -0,0 +1,75 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter_view/main.dart' as app;
+import 'package:integration_test/integration_test.dart';
+
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  group('end-to-end test', () {
+    testWidgets('flutter view to native view test counter',
+        (WidgetTester tester) async {
+
+      /*
+      The app draws itself. As the MyHomePage is initialized, the message
+      handler for "increment" is initialized in the `initState` override
+      for _MyHomePageState.
+      */
```

Comment:
Generally, prefer `//` comments in Dart code.

https://dart.dev/guides/language/effective-dart/documentation#dont-use-block-comments-for-documentation